spec.bs

```diff
@@ -1509,18 +1521,38 @@ To <dfn>generate and score bids</dfn> given an [=auction config=] |auctionConfig
     to the result of running [=is debugging only in cooldown or lockout=] with |buyer|.
   1. [=map/For each=] |signalsUrl| → |perSignalsUrlGenerator| of |perSlotSizeQueryParam|:
     1. [=map/For each=] |slotSizeQueryParam| → |perSlotSizeQueryParam| of |perBuyerGenerator|:
+      1. Let |allTrustedBiddingSignals| be an [=ordered map=].
       1. Let |keys| be a new [=ordered set=].
       1. Let |igNames| be a new [=ordered set=].
+      1. Let |lengthLimit| be the maximum value of {{long}} interger.
       1. Let |fetchSignalStartTime| be |settings|'s [=environment settings object/current monotonic time=].
       1. [=map/For each=] joiningOrigin → |groups| of |perSignalsUrlGenerator|:
         1. [=list/For each=] |ig| of |groups|:
-          1. [=set/Append=] |ig|'s [=interest group/trusted bidding signals keys=] to |keys|.
-          1. [=set/Append=] |ig|'s [=interest group/name=] to |igNames|.
-      1. Let |biddingSignalsUrl| be the result of [=building trusted bidding signals url=] with
-        |signalsUrl|, |keys|, |igNames|, |buyerExperimentGroupId|, |topLevelOrigin|, and
-        |slotSizeQueryParam|.
-      1. Let « |allTrustedBiddingSignals|, |dataVersion| » be the result of [=fetching trusted signals=]
-        with |biddingSignalsUrl| and true.
+          1. Set |keys| to |putativeKeys|.
+          1. Set |igNames| to |putativeIgNames|.
+          1. Set |lengthLimit| to |putativeLengthLimit|, if |lengthLimit| &le; |ig|'s
+            [=interest group/max trusted bidding signals url length=]. Otherwise, set
+            |ig|'s [=interest group/max trusted bidding signals url length=] to |putativeLengthLimit|.

```

Comment:
I feel you wanted to do the assignment oppositely? i.e., set |putativeLengthLimit| to |ig|'s [=interest group/max trusted bidding signals url length=].

Reply:
Yes, good catch